gradle.properties

```diff
@@ -1,6 +1,6 @@
-camelVersion = 4.3.0
-camelQuarkusVersion = 3.6.0
+camelVersion = 4.4.0
+camelQuarkusVersion = 3.7.3
```

Comment:
It looks like the latest version of the quarkus catalog is still 3.7.0

Reply:
Thanks I just spotted that too.